init.js

```diff
@@ -42,11 +42,11 @@ db.sequelize.sync({ force: true })
     return db.Role.create({ name: 'public' });
   })
   .then(function () {
-    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/'})
+    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/index'});
```

Comment:
It needs to be able to handle `/` as well. I would add that in as a part of this PR.